mitmproxy/addons/streambodies.py

```diff
@@ -15,8 +15,15 @@ def configure(self, updated):
             except ValueError as e:
                 raise exceptions.OptionsError(e)
 
-    def run(self, f, is_request):
+    def run(self, f, is_request, is_wsflow):
         if self.max_size:
+            if is_wsflow:
```

Comment:
IMO in this case it is cleaner to use `isinstance(f, WebSocketFlow)` instead of a new bool argument.